src/components/Forex.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import ShowButton from './ShowButton';
+import { categories } from './data';
+import '../styles/Forex.css';
+
+function Forex() {
+  const forex = useSelector((state) => state.forex);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [itemsToShow, setItemsToShow] = useState(3);
+  const navigate = useNavigate();
+
+  const show = () => {
+    if (itemsToShow === 3) {
+      setItemsToShow(forex.length);
+      setIsExpanded(true);
+    } else {
+      setItemsToShow(3);
+      setIsExpanded(false);
+    }
+  };
+
+  return (
+    <section className="forex-page">
+      <div className="top-bar">
+        <button
+          type="button"
+          className="go-back"
+          onClick={() => navigate('/')}
+        >
+          <i className="chevron left icon" />
+        </button>
```

Comment:
- [ ] As it is requirement for this project, which also improves the user experience of your app, kindly ensure  there's a link(`"Back" button`) in the `Detail page` that you can click on to go to the `Home page`. This button is absent in your app, please kindly fix that. See screenshot of the requirements below;

![backbutton](https://user-images.githubusercontent.com/65852621/152334021-f32e00b9-6377-448a-87f3-e1aa6b61a3e4.png)